### ISSUE TITLE
Isn't this what Yeoman tries to do?

### ISSUE THREAD

**Issue by white-gecko** [Dockerizing/NodeJS-NPM-ECMA6-Stack#3]
Isn't this what http://yeoman.io/ tries to do?

**Comment by GeorgesAlkhouri** [comment]
You're right. Yeoman is a nice tool for settings things up. But Yeoman is also just a npm module and therefore needs node to run. 
Our goal was to enable developing without caring about node version, npm version or any other OS environmental things.

**Comment by k00ni** [comment]
The idea behind that container is to provide a complete JavaScript environment incl. relevant technologies and tools. A developer doesn't have to setup and maintain NPM, NodeJS, which can be very tricky and needs effort to maintain. In this way you can reuse a maintained system, on which you can built your own project on. 

**In a nutshell:**
* Preconfigured and fixed version of major software such as Node.JS and NPM
* Possibility to set a configuration to initialize a project with certain JavaScript-modules (e.g. package.json)
* Usage of ECMA6 without the hassle to compile it by hand, because the container does it for you
* Predefined set of Linter-rules for good code style

Because of all the predefined stuff you make a suggestion/provide best practices how to develop with JavaScript. I think a lot of developers can benefit from that.
 
@white-gecko: Is Dockerizing the right place for that Docker? After thinking about we should probably move it to AKSW or such.

**Comment by k00ni** [comment]
I opened an issue about what this container is and can be: #5
